jekyll/_cci2/migrating-from-launch-agent-to-the-new-machine-runner-agent.adoc

```diff
@@ -0,0 +1,165 @@
+
+. Finally, move the agent configuration file. The default location for this is `/Library/Preferences/com.circleci.runner/launch-agent-config.yaml`. Move the file to `$HOME/Library/Preferences/com.circleci.runner/config.yaml`. 
+
+Alternatively you can let the Machine Agent install process create a config file template and copy the existing config into the newly created file (CROSSLINK)
+
+[#uninstalling-launch-agent-windows]
+=== Windows
+
+. To uninstall the launch agent, you can download the link:https://github.com/CircleCI-Public/runner-installation-files/blob/main/windows-install/Uninstall-CircleCIRunner.ps1[`Uninstall-CircleCIRunner.ps1` script] from GitHub to an easily accessible location. Then, run the script with the following command (from an administrator PowerShell console):
+
++
+```shell
+./Uninstall-CircleCIRunner.ps1
+```
+
++
+This will remove all launch agent components, including the configuration file.
+
++
+If you want to reuse the values with the new install, you should save the file to a temporary location while migrating. It is stored by default at `$env:ProgramFiles\CircleCI`
+
+
+[#install-machine-agent]
+== 2. Installing the Machine Agent
+
+Packaged versions of the machine runner agent are available for Linux and MacOS for a streamlined installation experience.
+
+[#install-machine-agent-linux]
+=== Linux
+
+. At present deb and rpm packages are available for linux. Use the following commands to install them
+
++
+```shell
+# DEB Commands
+# This script installs the package repository to the package manager
+curl -s https://packagecloud.io/install/repositories/circleci/runner/script.deb.sh | sudo bash
+sudo apt install circleci-runner
+```
+
++
+```shell
+# RPM Commands
+# This script installs the package repository to the package manager
+curl -s https://packagecloud.io/install/repositories/circleci/runner/script.rpm.sh | sudo bash
+sudo yum install circleci-runner
+```
+
++
+The packages will install the latest version of the agent. They also create a circleci user as well as required directories for operation of the agent.
+
+. The packages will create a template configuration file at `/etc/circleci-runner/circleci-runner-config.yaml`. This configuration is 1:1 compatible with the launch agent configuration, apart from the auto-update feature. Since auto-update has been removed from the new agent, you should use the appropriate package management update tools to get new versions of the package when they are released. When using the `apt` package to update the agent package, user interaction will be required to clear the configuration file modified prompt. This can be avoided using the `--force-confold` flag or by removing the modified configuration file before upgrading. 
```

Comment:
Can we just say "you will need to" rather than "user interaction will be required"

Reply:
Unless this is a different user to the one who is performing this migration, if so this can be left as is